src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsRulesSpec.groovy

```diff
@@ -988,4 +1000,139 @@ class PriceFloorsRulesSpec extends PriceFloorsBaseSpec {
         where:
         enforcePbs << [true, null]
     }
+
+    def "PBS should use correct rule for each bidder when bidder is define with valid bid floor value"() {
+        given: "PBS config with openX bidder"
+        def floorsPbsService = pbsServiceFactory.getService(
+                FLOORS_CONFIG + GENERIC_ALIAS_CONFIG +
+                        ["adapters.openx.enabled" : "true",
+                         "adapters.openx.endpoint": "$networkServiceContainer.rootUri/auction".toString()])
+
+        and: "BidRequest with generic and openx bidder"
```

Comment:
Test body sets `generic` to `null`, but description says that the request has it.

Reply:
This remark got moved for some reason, but it still stands.
```groovy
        and: "Default bid Request with generic and openx bidder within separate imps"
        def bidRequest = BidRequest.defaultBidRequest.tap {
            addImp(Imp.defaultImpression.tap {
                ext.prebid.bidder.generic = null
                ext.prebid.bidder.openx = Openx.defaultOpenx
            })
        }

```

Reply:
Oh, sorry, my derp. Wasn't attentive enough. Resolving.